README.rst

```diff
@@ -10,11 +10,11 @@ Why Should I Use This?
 ----------------------
 
 The biggest reason to use twine is that it securely authenticates you to PyPI
-over HTTPS using a verified connection while ``python setup.py upload`` `uses
-HTTP and exposes your credentials <http://bugs.python.org/issue12226>`_.  This
-means anytime you use it you expose your username and password to being
-sniffed. Twine uses only verified TLS to upload to PyPI protecting your
-credentials from theft.
+over HTTPS using a verified connection while ``python setup.py upload`` `only
+recently stopped using HTTP <http://bugs.python.org/issue12226>`_.  This means
+anytime you use it you expose your username and password to being sniffed.
```

Comment:
While you're here, could you update this line to read

    anytime you use it, you expose your username ...

(emphasis on the comma)